pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -184,11 +179,6 @@ func (ks *scaler) handleScaleToZero(pa *pav1alpha1.PodAutoscaler, desiredScale i
 	if pa.Status.IsActivating() { // Active=Unknown
 		// Don't scale-to-zero during activation
 		if min, _ := pa.ScaleBounds(); min == 0 {
-			if pa.Status.IsActivatingTimeout(config.ActivatingTimeout) {
```

Comment:
This code was the main code that did scale to zero, so it should've remained somewhere, right?